sdks/typescript/client-typescript-eventsource/README.md

```diff
@@ -9,7 +9,7 @@ guidelines.
 Below explains how you can use the FeatureHub SDK in Javascript or Typescript for applications like Node.js
 backend server or Web front-end (e.g. React). 
 
-This SDK is based on EventSource (or Server Sent Events) interface and provides real-time connection to the FeatureHub Edge server. 
+This repository includes the Server Sent Events realtime update SDK for Javascript and Typescript.
```

Comment:
Not only right? Now it also includes REST SDK... perhaps this sentence can be more generic

Reply:
no, it doesn't